teuthology/misc.py

```diff
@@ -647,7 +646,7 @@ def remove_lines_from_file(remote, path, line_is_valid_test,
     on when the main site goes up and down.
     """
     # read in the specified file
-    in_data = ensure_str(get_file(remote, path, False))
+    in_data = get_file(remote, path, False).decode()
```

Comment:
I'm thinking about adding another method `Remote.read_file()` for usability.

Reply:
BTW, read_file is a better name, IMHO.